### ISSUE TITLE
firefox 57

### ISSUE THREAD

**Issue by koullis22** [ewwink/X-Notifier-Reloaded#23]
so what now? do we stay in 56 or is there any way to make it  working on 57?

**Comment by Loirooriol** [comment]
If you install Firefox 57 Nightly or Developer Edition, then you will be able to enable legacy extensions. See https://wiki.mozilla.org/Add-ons/Firefox57#Prefs

But the add-on released by ewwink won't work properly, you will need to add [my patches](https://github.com/ewwink/X-Notifier-Reloaded/pulls/Loirooriol). For Firefox 58 see my fork for additional patches, but there is recent issue I haven't fixed yet.

**Comment by koullis22** [comment]
thank you for your reply...as i understand those 2 versions are more likely like tests and send feedback..
-is there a way to disable it?
-normal ff57 doesnt have those 2 options that need to work?
-as a normal ff user what would be better?

**Comment by Loirooriol** [comment]
No, in Firefox 57+ Release and Beta it's not possible to install legacy extensions, unless they are signed by Mozilla internally.

See https://www.mozilla.org/firefox/channel/desktop/
 - Firefox Nightly is updated everyday with the latest features, and thus is less stable. It's two versions ahead of Release.
 - Firefox Beta is more stable than Nightly but less than Release. It's one version ahead of Release.
 - Developer Edition is like beta but with some additional features for developers, like allowing legacy extensions.
 - There is also [ESR](https://www.mozilla.org/en-US/firefox/organizations/). Targeted for organizations, it's even more stable than Release.

So as a normal user you should probably choose one of

 - Use Firefox 52.x ESR, and stay in the 52.x branch even after Firefox 59.x ESR is released. Note the support for 52.x ESR will end when Firefox 61 is released, scheduled on 2018-07-03 (see [calendar](https://wiki.mozilla.org/RapidRelease/Calendar)). At that point you should choose another option.
 - Use Developer Edition and enable legacy extensions. But the extensions may require some fixes.
 - Replace Firefox with some fork which does not block legacy extensions (maybe Waterfox or Palemoon?). But the extensions may require some fixes.
 - Give up using legacy extensions.

**Comment by koullis22** [comment]
thanks for your clear answer:) and 1 last choice is just to install normal version 57 and install v4

**Comment by jeroen1956** [comment]
I agree: very clear explanation on all available options!
Oriol, can you post this on the forum?
Jeroen

2017-11-15 19:23 GMT+01:00 koullis22 <notifications@github.com>:

**Comment by Loirooriol** [comment]
> Oriol, can you post this on the forum?

Done! http://xnotifier.tobwithu.com/dp/node/3645

**Comment by jeroen1956** [comment]
Thanks!!

2017-11-15 22:14 GMT+01:00 Oriol Brufau <notifications@github.com>:
